docs/reference/doofinder.md

```diff
@@ -62,12 +63,12 @@
 |  [RawTermsInfo](./doofinder.rawtermsinfo.md) | RAW information for a set of terms belonging to a terms facet. |
 |  [RawTermStats](./doofinder.rawtermstats.md) | RAW information for a term belonging to a terms facet. |
 |  [RedirectionStatsParams](./doofinder.redirectionstatsparams.md) | Parameters for redirection stats. |
+|  [SearchImageParams](./doofinder.searchimageparams.md) | Set of params that are dump from a [Query](./doofinder.query.md)<!-- -->. |

```

Comment:
"Are dumped" might be better in this case

Reply:
done